packages/protocol/utils.ts

```diff
@@ -1,5 +1,6 @@
 import * as Sentry from "@sentry/browser";
-import { isDevelopment } from "ui/utils/environment";
+// TODO [bvaughn] protocol
+import { isDevelopment } from "shared/utils/environment";
```

Comment:
feels funny for the protocol to need to know about `environment`. If anything we could pass an environment in when creating a session.

Reply:
Yeah! Maybe so. Need to think about that more.

I don't think this (packages/protocol) is very close to what we'll eventually publish to NPM btw! The main goal of these PRs is to get things in a place where I can start prototyping something on top of the shared packages/protocol code. The "something" will, hopefully be the thing we eventually publish for others to build on top of.